From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tyler Veness <calcmogul@gmail.com>
Date: Tue, 28 Jan 2025 22:19:31 -0800
Subject: [PATCH 4/6] Replace std::to_underlying()

---
 src/util/PrintDiagnostics.hpp | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/util/PrintDiagnostics.hpp b/src/util/PrintDiagnostics.hpp
index 5124773d966143cd5b1f30d16f957b538d7cf401..093569223a20d954041cf4e4e3411fd4165b554d 100644
--- a/src/util/PrintDiagnostics.hpp
+++ b/src/util/PrintDiagnostics.hpp
@@ -9,7 +9,6 @@
 #include <cmath>
 #include <ranges>
 #include <string>
-#include <utility>
 
 #include <wpi/SmallVector.h>
 
@@ -58,8 +57,8 @@ void PrintIterationDiagnostics(int iterations, IterationMode mode,
 
   constexpr const char* kIterationModes[] = {" ", "s"};
   sleipnir::print("{:4}{}  {:9.3f}  {:13e}  {:13e}  {:13e}  ", iterations,
-                  kIterationModes[std::to_underlying(mode)], ToMs(time), error,
-                  cost, infeasibility);
+                  kIterationModes[static_cast<uint8_t>(mode)], ToMs(time),
+                  error, cost, infeasibility);
 
   // Print regularization
   if (δ == 0.0) {
