From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tyler Veness <calcmogul@gmail.com>
Date: Wed, 26 Jun 2024 12:13:33 -0700
Subject: [PATCH 5/6] Suppress clang-tidy false positives

---
 include/sleipnir/autodiff/Variable.hpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/sleipnir/autodiff/Variable.hpp b/include/sleipnir/autodiff/Variable.hpp
index d254bae5bdf13bc3e9705fd34d1647bc7ef3971b..2960b826aa0c2de3ab2edfd61332f53e993a229e 100644
--- a/include/sleipnir/autodiff/Variable.hpp
+++ b/include/sleipnir/autodiff/Variable.hpp
@@ -573,7 +573,7 @@ struct SLEIPNIR_DLLEXPORT EqualityConstraints {
    *
    * @param equalityConstraints The list of EqualityConstraints to concatenate.
    */
-  EqualityConstraints(
+  EqualityConstraints(  // NOLINT
       std::initializer_list<EqualityConstraints> equalityConstraints) {
     for (const auto& elem : equalityConstraints) {
       constraints.insert(constraints.end(), elem.constraints.begin(),
@@ -636,7 +636,7 @@ struct SLEIPNIR_DLLEXPORT InequalityConstraints {
    * @param inequalityConstraints The list of InequalityConstraints to
    * concatenate.
    */
-  InequalityConstraints(
+  InequalityConstraints(  // NOLINT
       std::initializer_list<InequalityConstraints> inequalityConstraints) {
     for (const auto& elem : inequalityConstraints) {
       constraints.insert(constraints.end(), elem.constraints.begin(),
